src/thing/Portal.cpp

```diff
@@ -50,13 +57,44 @@ void Portal::Start() {
           }
         }
       }
+      function scan() {
+        document.getElementById('scanning').style.display='inline';
+        var xhr = new XMLHttpRequest();
+        xhr.open("GET", "/wifi/scan", true);
+        xhr.onreadystatechange = function () {
+          if (xhr.readyState==4 && xhr.status==200) {
+           load_networks(JSON.parse(xhr.responseText));
+           document.getElementById('scanning').style.display='none';
+          }
+        }
+        xhr.send();
+      }
+      function load_networks(networks) {
+        select = document.getElementById('networks');
+        select.innerHtml = '';
+        networks.sort(function(a, b) {
+          return b.rssi - a.rssi;
```

Comment:
It seems like the sorting that would be most desirable, you get the strongest networks first. This is how OSX, Android, iOS, Windows, ChromeOS, etc.. sort.

The appeal of this sorting is that the most relevant networks are most likely to be on top. For example when I used to live in downtown LA I had roughly 20 networks I could see at a time but my home network was always the first or second because it was the strongest in my house.